### PR TITLE
Added simplified version of restamper script.

### PR DESCRIPTION
**Pull request by nikolicj**
Generates a bag file identical to an input bag, but with time stamps reassigned to input message header time stamps, as requested by @burrimi .

### CODE REVIEW

File: python/restamp_bag.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            outputfile = arg
+
+    # print console header
+    print ""
+    print "restamp_bag"
+    print ""
+    print 'input file:  ', inputfile
+    print 'output file: ', outputfile
+    print ""
+    print "starting synchronization (may take a while)"
```
Comment (rehderj) [inline python/restamp_bag.py]: Synchronization might not be the correct term for this.

File: python/restamp_bag.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            messageCounter = messageCounter + 1
+
+    # print console footer
+    finally:
+        print ""
+        print ""
+        print "finished iterating through input bag"
+        print "output bag written"
+        print ""
+        print "if errors occurred, please call +41 (0)44 632 99 65"
```
Comment (rehderj) [inline python/restamp_bag.py]: This does not appear to be a valid phone number. Please change.

File: python/restamp_bag.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            messageCounter = messageCounter + 1
+
+    # print console footer
+    finally:
+        print ""
+        print ""
+        print "finished iterating through input bag"
+        print "output bag written"
+        print ""
+        print "if errors occurred, please email joern.rehder@mavt.ethz.ch"
```
Comment (rehderj) [inline python/restamp_bag.py]: ...

Comment (burrimi) [inline python/restamp_bag.py]: bin mit jörn einverstanden, dass wir hier ev deine adresse angeben sollten ;)

**Comment by burrimi** [comment]
thanks a lot @nikolicj lgtm.

**Pull request merged**